api/RTCRtpSender.json

```diff
@@ -659,6 +659,351 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "active": {
+            "__compat": {
+              "description": "<code>parameters.encodings.active</code> parameter",
+              "support": {
+                "chrome": {
+                  "version_added": "69"
+                },
+                "chrome_android": "mirror",
+                "edge": {
+                  "version_added": "≤79"
+                },
+                "firefox": {
+                  "version_added": "110"
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "12.1"
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "dtx": {
+            "__compat": {
+              "description": "<code>parameters.encodings.dtx</code> parameter",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "12.1"
```

Comment:
Do I need to worry that this was set as `"version_added": "11"` in the dictionary? I.e. did you retest the support versions?

Reply:
https://github.com/WebKit/WebKit/commit/6cac337edef8bf68aede45c8c1e7d5ef325d0c27#diff-abbd2c5d74ae3a8ad02fe5deb809720da096c1b16bde6b039e7b24e7bf04f74c was first available in Safari 11. I don't know about flags, but unless we have other information I would leave this at Safari 11.